Library/Formula/fish.rb

```diff
@@ -41,6 +40,12 @@ def caveats; <<-EOS.undent
     to /etc/shells. Run:
       chsh -s #{HOMEBREW_PREFIX}/bin/fish
     to make fish your default shell.
+
+    To set your colors, run 'fish_config'
+    To scan your man pages for completions, run 'fish_update_completions'
+    To autocomplete command suggestions press Ctrl + F or right arrow key.
+
+    Have fun!
```

Comment:
We try not to add caveats for things that aren't Homebrew-specific; can you remove these?